admin/workspace-management/cvms/management.md

```diff
@@ -27,4 +32,56 @@ install `shiftfs` on your behalf.**
 
 > GPUs are not supported with cached CVMs at this time.
 
-![Cached CVMs](../../../assets/admin/cached-cvms.png)
+## Self-contained workspace builds
+
+> Self-contained workspace builds are currently an **alpha** feature.
+
+By default, Coder initializes workspaces by running commands inside the
+container. When you enable [self-contained workspace builds], workspaces control
+the initialization sequence instead. This enables operation on clusters that
+restrict command execution inside containers using the Kubernetes API, such as
+with the `kubectl exec` command.
+
+[self-contained workspace builds]: ../self-contained-builds.md
+
+## Workspace process logging
+
+> Workspace process logging is currently an **alpha** feature.
+
+[Workspace process logging] enables auditing of commands executed inside the
+workspace container.
+
+[workspace process logging]: ../process-logging.md
+
+## TUN device
+
+> TUN devices currently an **alpha** feature.
```

Comment:
@deansheather there didn't seem to be much actual docs in the separate TUN device doc, so I moved the docs here. WDYT?